src/09_dictionaries.py

```diff
@@ -36,12 +36,25 @@
 # Add a new waypoint to the list
 # YOUR CODE HERE
 
+waypoints.append({
+    "lat": 44,
+    "lon": -121,
+    "name": "a forth place"
+})
+print(waypoints)
 # Modify the dictionary with name "a place" such that its longitude
 # value is -130 and change its name to "not a real place"
 # Note: It's okay to access the dictionary using bracket notation on the
 # waypoints list.
 
 # YOUR CODE HERE
+for place in waypoints:
```

Comment:
If you know the index, this is also doable instead of a loop:

```
waypoints[0]["name"] = 'not a real place'
waypoints[0]["lon"] = -130
```
(I'd usually do it the way you did, but technically it's faster to access it directly.)